sdk/cognitivelanguage/Azure.AI.Language.Conversations/src/autorest.md

```diff
@@ -56,3 +56,143 @@ directive:
   where: $.parameters.Endpoint
   transform: $["format"] = "url"
 ```
+
+```yaml
+directive:
+- rename-operation:
+    from: AnalyzeConversation_SubmitJob
+    to: ConversationAnalysis_SubmitJob
+```
+
+```yaml
+directive:
+- rename-operation:
+    from: AnalyzeConversation_JobStatus
+    to: ConversationAnalysis_JobStatus
+```
+
+### Async API swagger mismatches
+
+```yaml
```

Comment:
Discussed in the chat 
/cc @mshaban-msft 

The directives now are necessary since there are a lot of mismatches. Will be reported to service team later.